lib/src/intl_suggestors/intl_configs_migrator.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:codemod/codemod.dart';
+import 'package:file/file.dart';
+import 'package:over_react_codemod/src/intl_suggestors/utils.dart';
+
+const varsToCheck = ['displayName', 'name', 'title'];
+
+class ConfigsMigrator extends RecursiveAstVisitor with AstVisitingSuggestor {
+  final String _className;
+  final File _outputFile;
+
+  ConfigsMigrator(this._className, this._outputFile);
+
+  @override
+  void visitMethodDeclaration(MethodDeclaration node) {
+    super.visitMethodDeclaration(node);
+    if (varsToCheck.contains(node.name.name)) {
+      if (node.body is ExpressionFunctionBody) {
```

Comment:
Does Dart do short-circuiting in boolean expressions? If so you could probably put these into one if statement since it doesn't look like anything is an exception to it.

Reply:
I just kept it simple.  I didn't want to end up with
```Dart
if (varsToCheck.contains(node.name.name) && node.body is ExpressionFunctionBody && (body as ExpressionFunctionBody).expression is SimpleStringLiteral) {
```